packages/contracts/contracts/core/IDAO.sol

```diff
@@ -13,121 +15,114 @@ abstract contract IDAO {
         bytes data; // FuncSig + arguments
     }
 
-    /// @notice Checks if an address has permission on a contract via a role identifier and considers if `ANY_ADDRESS` was used in the granting process.
-    /// @param _where The address of the contract
-    /// @param _who The address of a EOA or contract to give the permissions
-    /// @param _role The hash of the role identifier
-    /// @param _data The optional data passed to the ACLOracle registered
-    /// @return bool Returns whether the address has permission is or not
-    function hasPermission(
+    /// @notice Checks if an address has permission on a contract via a permissionID identifier and considers if `ANY_ADDRESS` was used in the granting process.
```

Comment:
Let's add the `` around `permissionId`  so it's clearer we're referring to that variable.

Reply:
Done